.gitignore

```diff
@@ -73,3 +73,5 @@ pm2.json
 settings.json
 build.sh
 /public/livechat
+.screenshots
+packages/rocketchat-google-vision/.npm/package/npm-shrinkwrap.json
```

Comment:
does that actually work? I tried that earlier but it was still appearing in the diff

Reply:
the .screenshots seems to work the second one, unfortunately, doesn't